### ISSUE TITLE
Systray menu shows boxes instead of text in linux

### ISSUE THREAD

**Issue by ikirudennis** [ytmdesktop/ytmdesktop#377]
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**.
- [X] I verified that there is no open issue for the same subject.

**Describe the bug**
The systray menu shows boxes instead of text. Seems like the font isn't loading properly. This was working properly in 1.12

**To Reproduce**
1. Open ytmdesktop
2. Click on systray icon
3. See boxes instead of text

**Expected behavior**
3. See text of the menu

**Screenshots**
![ytm-desktop_menu](https://user-images.githubusercontent.com/1294347/92941120-f6cbc600-f41d-11ea-8871-390b758c244b.png)

**Environment (please complete the following information):**
 * YTMDesktop version: ***1.13.0***
 * OS: ***Linux***
 * OS version: ***Fedora 32*** Running tint2 for the system tray in openbox desktop environment
 * Arch: ***x64***
 * Installation way: ***snapcraft***

**Comment by adlerluiz** [comment]
after restarting, does the problem still occur?

**Comment by ikirudennis** [comment]
Unfortunately, yes it persists across restarts. I also tested it with and without the `/snap` symlink, but it doesn't work either way.
Some other items to note: On starting, the systray icon is blank, but when I start playing something it then shows up. Also, it frequently crashes on exit. I'm not sure if any of that is helpful information.

**Comment by nishp77** [comment]
Hi guys, i would like to add a feature to the system tray, that is could have a GUI like maybe on hover

**Comment by ikirudennis** [comment]
It seems like it might be this problem: https://forum.snapcraft.io/t/snapped-app-not-loading-fonts-on-fedora-and-arch/12484 Unfortunately, I haven't had luck with any of the suggestions therein.

**Comment by psiryan** [comment]
> It seems like it might be this problem: https://forum.snapcraft.io/t/snapped-app-not-loading-fonts-on-fedora-and-arch/12484 Unfortunately, I haven't had luck with any of the suggestions therein.
I agree; having the same issue on Arch. Appears from the linked post that this issue affects Fedora and Arch.

**Comment by thwaller** [comment]
It has been a while on this, but wanted to report having this issue. Just confirming the issue still exists, no change. Fedora 36.

**Comment by Alipoodle** [comment]
Would love to know if this is still an issue on YTM Desktop V2

**Comment by jippylong12** [comment]
This might be resolved in v2 since the project no longer distributes via Snap. The issue appears to be a [known Snap confinement problem](https://forum.snapcraft.io/t/snapped-app-not-loading-fonts-on-fedora-and-arch/12484) where sandboxed apps can't access system fonts on Fedora and Arch.

v2 packages as `.deb` and `.rpm` instead, which should have full access to system fonts. If anyone is still seeing this with v2 using the RPM or DEB package, it would be worth opening a new issue with updated details.

Could probably be closed as no longer applicable.
